### PR TITLE
feat: implement cchecksum for ~2x faster checksumming

### PR DESCRIPTION
**Pull request by BobTheBuidler**
### What I did

fixes: #

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by antazoey** [comment]
question: would it possible to update the one in eth-utils instead?

**Comment by BobTheBuidler** [comment]
Not sure yet. I opened a PR on the web3.py repo and if that's the direction they want me to swim in I will. Until then, I decided it best to leave the cython implementation fully separate.

File: setup.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
     url="https://github.com/ApeWorX/eth-pydantic-types",
     include_package_data=True,
     install_requires=[
+        "cchecksum>=0.0.3,<1"
```
Comment (antazoey) [inline setup.py:77]:
```suggestion
        "cchecksum>=0.0.3,<1,"
```

File: setup.py [hunk 2]
```diff
@@ -74,6 +74,7 @@
     url="https://github.com/ApeWorX/eth-pydantic-types",
     include_package_data=True,
     install_requires=[
+        "cchecksum>=0.0.3,<1,"
```
Comment (antazoey) [inline setup.py:77]:
```suggestion
        "cchecksum>=0.0.3,<1",
```

File: eth_pydantic_types/address.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from typing import Annotated, Any, ClassVar, Optional, cast
```
Comment (antazoey) [inline eth_pydantic_types/address.py:1]:
```suggestion
from typing import Annotated, Any, ClassVar, Optional
```

**Review by antazoey — APPROVED** [review]

**Review by antazoey — APPROVED** [review]

**Pull request merged**